lib/assets/javascripts/cartodb/public_common/sample_settings_view.js

```diff
@@ -68,7 +70,102 @@ module.exports = cdb.core.View.extend({
       return collection.fetch();
   },
 
+  _sampleMapSave: function() {
+    if(true) {
+      this._saveAsMap();
+    }
+    else {
+      this._saveMap();
+    }
+  },
+
+  _printExportState: function() {
+    var state = this._exportMapModel.get('state');
+    console.log('Export state: ' + state);
```

Comment:
The current production code has no console logging aside from the asset and cartodb.js versions - I think we should stick to that convention and remove this output.

Reply:
Agreed, will need to make one last pass before all this is merged to remove the debugging trace.  A few more changes will be needed before this can be tested as "production ready".